contracts/voucher/interfaces/IVoucher.sol

```diff
@@ -9,12 +9,15 @@ import { IUpgradeable } from "../../utility/interfaces/IUpgradeable.sol";
  * @dev Voucher interface
  */
 interface IVoucher is IUpgradeable, IERC721Upgradeable {
```

Comment:
It may be useful to expose a `controller()` public getter (but it's up to you)

Reply:
added